vignettes/tlf-kmplot.rmd

```diff
@@ -27,45 +27,45 @@ devtools::load_all()
 # Step 1: Read in data
 
 ```{r}
-adsl <- read_sas(file.path(path$adam, "adsl.sas7bdat"))
-adtte <- read_sas(file.path(path$adam, "adtte.sas7bdat")) 
+adsl <- read_xpt(file.path(path$adam, "adsl.xpt"))
+adtte <- read_xpt(file.path(path$adam, "adtte.xpt")) 
 ```
 
 ## Step 2: Data preparation for endpoint TTDE
 ```{r}
 anl <- adsl %>% 
   dplyr::filter(
-    ITTFL == "Y",
+    SAFFL == "Y",
     STUDYID == "CDISCPILOT01"
   ) %>%
-  dplyr::select(STUDYID, USUBJID, ARM) %>%
+  dplyr::select(STUDYID, USUBJID, TRT01A) %>%
```

Comment:
Let's only use TRT01P

Reply:
@elong0527 
For safety of interest TTDE, should we use planned treatment `TRT01P` or actual treatment `TRT01A`? Thanks!

Reply:
For safety, you are correct to use `TRT01A`.